### ISSUE TITLE
Update Contact list on ONRR.GOV

### ISSUE THREAD

**Issue by Sheetsj** [ONRR/onrr.gov-site#362]
Hello,

I need to update the contact list for Accounts Receivable, Billing, and Financial Issues Information-Indian. 

It is located on ONRR.GOV under Contact US>Online Resources > ONRR COntacts by Company Assignment>Accounts Receivable, Billing, and Financial Issues Information-Indian

The file that needs to be used is located: \\ISEDENNAS\DEN_Common\FMD\FinancialServices\Admin\Ind Assignment Contact List

File name is finserAssign.pdf Which I also attached to this message. 
[finserAssign.pdf.pdf](https://github.com/ONRR/onrr.gov-site/files/5555877/finserAssign.pdf.pdf)

Thank you for your help in this matter!
Jessica

**Comment by lpgoldstein** [comment]
@Sheetsj I have attached your accessibility report here. Rather than trying to fix the errors, we highly recommend creating the table in Excel and posting the Excel file to the website instead. You are able to check for accessibility issues within Excel itself and the format is more easily accessible than PDFs. 
[finserassign Accessibility Report.pdf](https://github.com/ONRR/onrr.gov-site/files/5555969/finserassign.Accessibility.Report.pdf)

As previously communicated by ONRR’s Deputy Director, as of September 1, 2020, electronic documents submitted for posting on onrr.gov must be accessible to employees and members of the public with disabilities.

The pdf document you have requested to be uploaded on onrr.gov aren't accessible and don't meet the 508 compliance standards. Attached are the accessibility report of the identified accessibility errors that must be corrected by the document owner before the Data Display team is able to upload the document on onrr.gov.

Section 508 of the Rehabilitation Act of 1973, as amended, requires that all Information and Communication Technology developed, procured, maintained, or used by federal agencies must be fully accessible to employees and members of the public with disabilities.

Data Display has created a list of resources and available training to assist with this process. If you have questions, please let us know.

**Comment by Sheetsj** [comment]
I am a little confused. This is how we were told to do these updates to the contacts. The ones we have on there now are pdf. Since if it was excel the companies could change these contact and that is not what we would want.

**Comment by lpgoldstein** [comment]
@Sheetsj I have to hop into a meeting in a few minutes. Can I give you a call in 45 minutes or so?

**Comment by Sheetsj** [comment]
I normally get off at 2:30 I can stay as late as 3:20.

**Comment by Sheetsj** [comment]
I just did this same thing on ticket #295 and it was not an issue. This was done 9/28/2020

**Comment by Sheetsj** [comment]
Okay I redid the file using one that I used with the other update. 
[finserAssign.pdf](https://github.com/ONRR/onrr.gov-site/files/5556285/finserAssign.pdf)

**Comment by lpgoldstein** [comment]
@Sheetsj This pdf looks great and is ready for your review: http://2017onrr/About/contact.htm

**Comment by Sheetsj** [comment]
@lpgoldstein I took a look at the link and it looks great thank you so much for your help with this. I will make sure that we always use the correct form from now on. Have a great day!

**Comment by lpgoldstein** [comment]
@Sheetsj It should be live soon. Thanks so much for knowing how to fix the accessibility issues. Please share your template with any colleagues that might need it since we are in the process of correcting all of the pdfs on the site to make them accessible.
